src/journal/journald-rate-limit.c

```diff
@@ -226,11 +217,18 @@ int journal_rate_limit_test(JournalRateLimit *r, const char *id, int priority, u
                         break;
 
         if (!g) {
-                g = journal_rate_limit_group_new(r, id, ts);
+                g = journal_rate_limit_group_new(r, id, rl_interval, ts);
                 if (!g)
                         return -ENOMEM;
+        } else {
+            g->interval = rl_interval;
         }
```

Comment:
coding style: please stick to multiples of 8ch. also, no {} around single-line if blocks. Also see CODING_STYLE